package amber.roboclaw_proto;
option java_package = "pl.edu.agh.amber.amber.roboclaw.proto";
option java_outer_classname = "RoboclawProto";

import "drivermsg.proto";

extend amber.DriverMsg {
    optional MotorsSpeed motorsCommand = 10;
    optional bool currentSpeedRequest = 11;
    optional MotorsSpeed currentSpeed = 12;
}

message MotorsSpeed {
    optional int32 frontLeftSpeed = 1;
    optional int32 frontRightSpeed = 2;
    optional int32 rearLeftSpeed = 3;
    optional int32 rearRightSpeed = 4;
}